index.html

```diff
@@ -16,6 +16,9 @@
       <section class="primary">
         <div class="container">
           <h1 class="page-title">Contrast Ratio Calculator</h1>
+          <p class="description">
+            Calculate the contrast ratio of text and background colors.
```

Comment:
The thing is: lower on the page we are using:
Foreground color and Background color. I mean the text above the inputs.

You wrote here
`Calculate the contrast ratio of text and background colors.`

Change it like
`Calculate the contrast ratio of foreground and background colors.`
or
`Calculate the contrast ratio of two colors`
or maybe even
`Find out contrast-ratio for two colors`